Gemfile

```diff
@@ -0,0 +1,4 @@
+source 'https://rubygems.org'
+
+gem 'rspec'
+gem 'pry'
```

Comment:
It's a silly thing, but just to get it out of the way - people will tell you to get the final newline character on there.